kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosApp.vue

```diff
@@ -281,10 +283,23 @@ export default {
         }
         this.$refs.kudosAPI.init();
 
-        document.addEventListener('exo-kudos-open-send-modal', this.openDialog);
+        document.addEventListener('exo-kudos-open-send-modal', this.openDrawer);
         document.addEventListener('exo-kudos-open-kudos-list', this.openListDialog);
       });
   },
+  computed: {
+    kudosReceiver () {
+      return {
+        receiverId: this.kudosToSend && this.kudosToSend.id,
+        avatar: this.kudosToSend && this.kudosToSend.avatar,
+        profileUrl: this.kudosToSend && this.kudosToSend.profileUrl,
+        fullName: this.kudosToSend && this.kudosToSend.receiverFullName
+      };
+    },
+    kudosSent () {
+      return 3 - this.remainingKudos;
```

Comment:
We should use the dynamic variable `window.kudosSettings.kudosPerPeriod` for total allowed kudos.